pkg/popeye.go

```diff
@@ -256,9 +256,34 @@ func (p *Popeye) Sanitize() (int, int, error) {
 			if err != nil {
 				log.Fatal().Err(err).Msg("Parse S3 bucket URI")
 			}
+
+			// Initialize the AWS session
+			var config *aws.Config
+
+			if isSetStr(p.flags.S3Region) && isSetStr(p.flags.S3Endpoint) {
```

Comment:
Not keen on this if/else extravaganza... imho, this could be simplified as the aws.Config handles the string pointers so we don't care if user set it or not ie something like this should suffice:
```go
			s, err := session.NewSession(&aws.Config{
				LogLevel: aws.LogLevel(aws.LogDebugWithRequestErrors),
				Region:   p.flags.S3Region,
				Endpoint: p.flags.S3Endpoint,
			})
```

Reply:
I removed the if/else statements